synapse/lib/stormtypes.py

```diff
@@ -7382,7 +7382,10 @@ class View(Prim):
                     The parent View iden.
 
                 nomerge (bool)
-                    Setting to $lib.true will prevent the layer from being merged.
+                    Deprecated - use protected. Setting to $lib.true will prevent the layer from being merged.
```

Comment:
Update the doc to reflect that reads/edits of nomerge will redirect to protected and omit the old behavior doc?